airtime_mvc/application/controllers/downgrade_sql/airtime_3.0.0-alpha.7/downgrade.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE cc_block ALTER COLUMN type SET DEFAULT 'static';
```

Comment:
Doesn't this also need an Upgrader class for schema version `3.0.0-alpha.7`? Like one of these:
https://github.com/LibreTime/libretime/blob/5b822afe14046157a8d9fbdff103eb383673e6be/airtime_mvc/application/upgrade/Upgrades.php#L523-L534
I think it won't automatically get applied without one but I'm not sure.

Reply:
I'm not sure. It probably does, I didn't test the upgrade directly. I just pushed the new Upgrader class as a PR.